app/controllers/api/users.php

```diff
@@ -2275,6 +2275,8 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
         $clone = clone $user;
 
         $dbForProject->deleteDocument('users', $userId);
+        \Appwrite\Deletes\Identities::delete($dbForProject, Query::equal('userInternalId', [$user->getInternalId()]));
```

Comment:
lets import \Appwrite\Deletes and use `Identities::delete` in the worker as well